openweathermap-ios/openweathermap-ios/BookmarkLocation/Model/BookmarkLocation.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  BookmarkLocation.swift
+//  openweathermap-ios
+//
+//  Created by Cassio Sousa on 24/04/20.
+//
+
+import Foundation
+
+private enum BookmarkLocationKeys: String {
```

Comment:
Embed this enum into an extension from BookmarkLocation.

Reply:
Fixed in b6bb083